Documentation/building/linux-instructions.md

```diff
@@ -1,14 +1,61 @@
 Build CoreCLR on Linux
 ======================
 
-This guide will walk you through building CoreCLR on Linux.  We'll start by showing how to set up your environment from scratch.
+This guide will walk you through building CoreCLR on Linux.  Before building there is environment setup that needs to happen to pull in all the dependencies required by the build.  There are two suggested ways to go about doing this. First you are able to use the docker environments provided by https://github.com/dotnet/dotnet-buildtools-prereqs-docker, or you can set up the environment yourself. The documentation will go over both ways of building; however, in general prefer docker if you are able. 
+
+[Build using docker](#Build-using-docker)
+
+[Build with own environment](#Environment)
+
+Build using docker
+==================
+
+Install Docker, see https://docs.docker.com/install/linux/docker-ce/ubuntu/ 
+
+Building using docker will require that you choose the correct image for your environment. Note that the OS is strictly speaking not extremely important, for example if you are on Ubuntu 18.04 and build using the Ubuntu 16.04 x64 image there should be no issues. The target arch is more important, as building arm32 using the x64 image will not work, there will be missing rootfs components required by the build. See [Docker Images](#Docker-Images) for more information on choosing an image to build with.
+
+Once you have chosen an image the build is one command run from the root of the coreclr repository:
+
+```sh
+docker run --rm -v <path/to/coreclr/repo>:/coreclr -w /coreclr microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-c103199-20180628134544 ./build.sh
+```
```

Comment:
It's also useful to show a completely expanded example, that doesn't contain any "replacement parameters" like `<path/to/coreclr/repo>` here.